stable/traefik/values.yaml

```diff
@@ -34,6 +34,13 @@ acme:
   email: admin@example.com
   staging: true
   logging: false
+  ## Enable httpChallenge http://v1-5.archive.docs.traefik.io/configuration/acme/#acmehttpchallenge
+  ## Only available in Traefik >=1.5.
+  ## To use it change:
+  ##  imageTag: 1.5
```

Comment:
Wouldn't it be better to just bump the Traefik version that this chart uses by default?

Reply:
Yes, the thing is that when I created this PR 1.5 was still a RC, but it was relaeased 2 day ago. Let me do those changes,